src/openloco/widget.cpp

```diff
@@ -449,7 +449,7 @@ namespace openloco::ui::widget
     // 0x004CA750
     void draw_23_caption(gfx::drawpixelinfo_t* dpi, window* window, widget_t* widget, uint16_t flags, uint8_t colour)
     {
-        stringFormatBuffer[0] = (uint8_t)144;
+        stringFormatBuffer[0] = (uint8_t)144; // black
```

Comment:
Should probably go into formatting constant instead; same for the ones in `draw_24` and `draw_25` below.